config/newrelic.yml

```diff
@@ -0,0 +1 @@
+license_key: <%= ENV.fetch("NEWRELIC_LICENSE_KEY") { "" } %>
```

Comment:
have you provided NEWRELIC_LICENSE_KEY to heroku application?

Reply:
yes